src/package.json

```diff
@@ -33,6 +33,7 @@
     "rollup-plugin-commonjs": "^10.1.0",
     "rollup-plugin-glslify": "^1.2.0",
     "rollup-plugin-node-resolve": "^5.2.0",
+    "rollup-plugin-scss": "^2.6.1",
```

Comment:
Do we need this plugin in the npm package?

Reply:
i don't think so, only in the example

Reply:
This seems to be in the wrong package.json. It is missing in the example